code/widget-showcase/canvas.tape

```diff
@@ -0,0 +1,9 @@
+# A VHS tape. See https://github.com/charmbracelet/vhs
+Output "../../src/content/docs/showcase/widgets/canvas.gif"
+Set Theme "Aardvark Blue"
+Set Width 600
+Set Height 400
```

Comment:
Nit: On the website, this renders a bit narrower than the rest of the widgets.

Reply:
That's what I found was the most balanced in term of empty/drawn space.
But let me take another look see if I can have something better.

Reply:
I found something that fits, pushed.